src/clj/fluree/db/virtual_graph/proto.cljc

```diff
@@ -0,0 +1,7 @@
+(ns fluree.db.virtual-graph.proto)
```

Comment:
I'm wary of protocol only namespaces because they allow cyclic dependencies to develop. Why can't this protocol live in the `fluree.db.virtual-graph.index-graph` namespace?